### DISCUSSION TITLE
Preview в темной теме

### DISCUSSION THREAD

**Discussion by Spyker086** [totumonline/totum-ru-issues-and-discussions#512] [ОШБК!] [answered]
Добрый день!
Заметил ошибку: в темной теме в полях "селект" превью выдается в белом поле с белым текстом.
![333](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/80567813/feffaae5-60e6-4d10-963d-597dd4dd0e68)

**Comment by ttm-alexeypolunin** [comment]
Да, надо поправить

**Reply by ttm-alexeypolunin** [reply, marked as answer]
Исправили V5
